### PR TITLE
RHDEVDOCS-6238: [DOC] Import jib-maven task

### PR DESCRIPTION
**Pull request by eromanova97**
Version(s): `pipelines-docs-1.17`

Issue: [RHDEVDOCS-62138](https://issues.redhat.com/browse/RHDEVDOCS-6283)

Link to docs preview:  https://85280--ocpdocs-pr.netlify.app/openshift-pipelines/latest/create/remote-pipelines-tasks-resolvers.html#op-taskref-jib-maven_remote-pipelines-tasks-resolvers

QE review:
- [ ] QE has approved this change.

**Additional information:**

### CODE REVIEW

File: modules/op-resolver-cluster-tasks-ref.adoc [hunk 1]
```diff
@@ -257,6 +257,69 @@ spec:
 * All result names were changed to uppercase.
 * The `gitInitImage` parameter was removed.
 
+[discrete]
+[id="op-taskref-jib-maven_{context}"]
+== jib-maven
+
+The `jib-maven` task builds Java, Kotlin, Groovy, and Scala source into a container image using the Jib tool. Jib works with Maven and Gradle projects. The following template is for Maven projects.
```
Comment (eromanova97) [inline modules/op-resolver-cluster-tasks-ref.adoc:264]: I took this line from the yaml file, but I am not sure about the support downstream, @aneeshmbhat could you please let me know?? Thank you!

File: modules/op-resolver-cluster-tasks-ref.adoc [hunk 2]
```diff
@@ -257,6 +257,69 @@ spec:
...
+.Results that the `jib-maven` task returns
+[cols="1,1,2",options="header"]
+|===
+| Result | Type | Description
+|`IMAGE_DIGEST` |`string` |Digest of the image that was built.
+|===
+
+.Changes from the `jib-maven` `ClusterTask`
+
+* TBD
```
Comment (eromanova97) [inline modules/op-resolver-cluster-tasks-ref.adoc:321]: @aneeshmbhat do we want to have something in this section here for users updating from cluster task to this task?

File: modules/op-resolver-cluster-tasks-ref.adoc [hunk 3]
```diff
@@ -257,6 +257,68 @@ spec:
 * All result names were changed to uppercase.
 * The `gitInitImage` parameter was removed.
 
+[discrete]
+[id="op-taskref-jib-maven_{context}"]
+== jib-maven
+
+The `jib-maven` task builds Java, Kotlin, Groovy, and Scala source into a container image using the Jib tool that works with Maven projects.
```
Comment (eromanova97) [inline modules/op-resolver-cluster-tasks-ref.adoc:264]: I took this line from the yaml file, but I am not sure about the support downstream, @aneeshmbhat could you please let me know?? Thank you!

File: modules/op-resolver-cluster-tasks-ref.adoc [hunk 4]
```diff
@@ -257,6 +257,65 @@ spec:
 * All result names were changed to uppercase.
 * The `gitInitImage` parameter was removed.
 
+[discrete]
+[id="op-taskref-jib-maven_{context}"]
+== jib-maven
+
+The `jib-maven` task builds Java, Kotlin, Groovy, and Scala sources into a container image by using the Jib tool for Maven projects. The Jib tool then pushes the image to the specified registry and provides an image digest for reference.
```
Comment (eromanova97) [inline modules/op-resolver-cluster-tasks-ref.adoc:264]: I took this line from the yaml file, but I am not sure about the support downstream, @aneeshmbhat could you please let me know?? Thank you!

File: modules/op-resolver-cluster-tasks-ref.adoc [hunk 5]
```diff
@@ -257,6 +257,65 @@ spec:
...
+|`sslcertdirt` |The optional workspace that contains SSL certificates.
+|`dockerconfig` |The optional workspace that contains the `docker` configuration.
+|===
+
+.Results that the `jib-maven` task returns
+[cols="1,1,2",options="header"]
+|===
+| Result | Type | Description
+|`IMAGE_DIGEST` |`string` |Digest of the image that was built.
+|===
```
Comment (eromanova97) [inline modules/op-resolver-cluster-tasks-ref.adoc:317]: @aneeshmbhat do we want to have something in this section regarding "changes from the `jib-maven` `ClusterTask` " here, similarly to the other examples,  for users updating from cluster task to this task?

**Comment by mramendi** [comment]
@savitaashture FYI - doc for jib-maven

**Pull request closed without merging**